java/src/test/java/com/google/protobuf/ParseExceptionsTest.java

```diff
@@ -0,0 +1,211 @@
+package com.google.protobuf;
```

Comment:
Doh! Good catch.

There were no conflicts after rebase and `mvn test` ran clean, so I totally didn't notice how this didn't fit with the rearranged modules. Sorry!

(Fixed now)